sdk/src/services/BackendWalletService.ts

```diff
@@ -665,6 +665,7 @@ export class BackendWalletService {
         requestBody: {
             message: string;
             isBytes?: boolean;
+            chainId?: number;
```

Comment:
Note from our convo: can you check if signing on different chains result in the same output. If so, we could default to a testnet (Sepolia?).